misc.lua

```diff
@@ -51,4 +54,39 @@ function flib_misc.ticks_to_timestring(tick)
   end
 end
 
+function flib_misc.get_signal_value(entity, signal)
+  local red = entity.get_circuit_network(wire_type.red)
+  local green = entity.get_circuit_network(wire_type.green)
+  local value = 0
+  if red then
+    value = red.get_signal_value(signal)
+  end
+  if green then
+    value = value + green.get_signal_value(signal)
+  end
+  return value
+end
+
+function flib_misc.check_signal(signal, blacklist_table)
```

Comment:
How useful is checking if a signal is valid and not blacklisted?
You can only get invalid signal prototypes from global or other mods.

Reply:
This is pretty useful for everyone that stores signal values in global. When a mod get removed, it can cause massiv errors. So this prevents it.